.gitignore

```diff
@@ -14,7 +14,7 @@
 
 # PHP-CS-Fixer
 /.php-cs-fixer.php
-/.php-cs-fixer.cache
+/.php-cs-fixer.cache/
```

Comment:
This seems to be unrelated.

Reply:
> chore: indicate that entry is dir and not regular file

I was fooled that there is a cache file, till I realised it's actually a dir. And as .gitignore allows to specify that entry is a dir, I use it when applicable.